src/SweetAlert/SweetAlertNotifier.php

```diff
@@ -223,4 +223,20 @@ private function hasTitle()
     {
         return (bool) strlen($this->config['title']);
     }
+    
+    /**
+     * Make Message HTML view.
+     * 
+     * @param bool|true $html
+     *
+     * @return \UxWeb\SweetAlert\SweetAlertNotifier $this
+     */
+    public function html($html = true)
```

Comment:
Hi @uxweb 
Thank for replay 

sorry i think i missed it while testing on my local machine. am going to update